public/kuroco_front.json

```diff
@@ -0,0 +1,25 @@
+{
+    "basic": [],
+    "ip_restrictions": [],
+    "redirects": [],
+    "rewrites": [
+      {
+        "source": ".*",
+        "destination": "/index.html"
+      }
+    ],
+    "inject_data": {
```

Comment:
@VortexExpansion 
I think it's not necessary if you can control meta data over `/reminder/`.
If not, please add it and consider the others (Login,...)

Reply:
@yabe-diverta san
Added separate Meta tags for reminder page and removed injection from kuroco front. 
As discussed, meta data for other pages not required at the moment. 
Resolved.